js/queen.js

```diff
@@ -38,12 +38,17 @@ export class Queen extends Piece{
 
         let maxRows = board.length;
         let maxCols = board[0].length;
-        let minRows, minCols = 0;
+        let minRows = 0;
+        let minCols = 0;
         /**
          * Get row increasing moves
          */
         for (let i = this.rowPos + 1; i < maxRows; i++) {
-            if (board[i][this.colPos] !== undefined && board[i][this.colPos].getPlayerId() == this.playerId) {
+            if (board[i][this.colPos] && board[i][this.colPos].getPlayerId() == this.playerId) {
```

Comment:
`===` should be used instead of `==` on line `47`.

Reply:
Updated.